search/categories_cache.cpp

```diff
@@ -31,14 +31,35 @@ CBV CategoriesCache::Get(MwmContext const & context)
   return cbv;
 }
 
+CBV CategoriesCache::GetFuzzy(MwmContext const & context)
+{
+  if (!context.m_handle.IsAlive() || !context.m_value.HasSearchIndex())
```

Comment:
We have an agreement that context is always valid here, no no need in these m_handle.IsAlive(). Also, no need in check that context has search index.

Reply:
I'm not sure how it is bad to be on the safer side.
That's old code and I don't see the agreement in its docs. Do you think I shall write it and replace the "if" with asserts?

Reply:
done